lib/pf/radius.pm

```diff
@@ -281,6 +281,28 @@ sub accounting {
         if($isStop){
             #handle radius floating devices
             $this->_handleAccountingFloatingDevices($switch, $mac, $port);
+            #unreg on disconnect
+            use Date::Parse;
+            my $node_info = node_view($mac);
+            my $time = time();
+            my $regdate = $node_info->{'regdate'};
+            my $convertregdate = str2time($regdate);
+            my $interval = $Config{'advanced'}{'unreg_on_disconnect_timing'};
+            my $limitunreg = $convertregdate + $interval;
+            my $filter = new pf::vlan::filter;
+            my $switch = pf::SwitchFactory->instantiate($node_info->{'last_switch'});
```

Comment:
Use the current value in the radius request because $node_info->{'last_switch'} come from the request before.
Same for ssid, connection_type,ifIndex